paper.tex

```diff
@@ -344,8 +345,8 @@ \section*{The ARG as a data structure}
 the data structure interpretation.
 
 The Griffiths ARG encoding is an elegant and mathematically
-economical description of the coalescent with recombination, but
-it has significant drawbacks as the basis for a general purpose
+economical description of the coalescent with recombination on a single
```

Comment:
This implies we're talking about a sample of size 1 to me - what are you trying to clarify here?

Reply:
Oh, I see. I mean that the Griffiths model only applies to an organism with a single chromosome. That's actually a much greater restriction than the one about multiple breakpoints, because most (all?) organisms have multiple chromosomes, so the inability to represent these is a significant deviation from the ability to describe the real world.